autoscheduler/scraper/serializers.py

```diff
@@ -1,7 +1,12 @@
+from datetime import time
 from rest_framework import serializers
 from .models.course import Course
 from .models.section import Section, Meeting
```

Comment:
```suggestion
from scraper.models import Course, Meeting, Section
```
This is my fault, but this should be changed to be consistent with the model imports in other files